introduction.md

```diff
@@ -1,2 +1,5 @@
 Hello, my name is Tony. I am working on the git master course in code school.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?